cli/utils.go

```diff
@@ -0,0 +1,191 @@
+package main
+
+import (
+	"context"
+	"crypto/ecdsa"
+	"fmt"
+	"log"
+	"math"
+	"math/big"
+	"os"
+
+	eigenpodproofs "github.com/Layr-Labs/eigenpod-proofs-generation"
+	"github.com/Layr-Labs/eigenpod-proofs-generation/cli/onchain"
+	"github.com/attestantio/go-eth2-client/spec/phase0"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	gethCommon "github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/fatih/color"
+)
+
+const (
+	ValidatorStatusInactive  = 0
+	ValidatorStatusActive    = 1
+	ValidatorStatusWithdrawn = 2
+)
+
+func PanicOnError(message string, err error) {
+	if err != nil {
+		color.Red(fmt.Sprintf("error: %s\n\n", message))
+
+		info := color.New(color.FgBlack, color.Italic)
+		info.Printf(fmt.Sprintf("caused by: %s\n", err))
+
+		os.Exit(1)
+	}
+}
+
+type ValidatorWithIndex = struct {
+	Validator *phase0.Validator
+	Index     uint64
+}
+
+type Owner = struct {
```

Comment:
`type Owner struct` - or is there something Im missing with the ` = struct`?

Reply:
just my brain writing typescript in Go

Go syntax is forgiving enough to allow both